moveit_ros/moveit_servo/test/launch/servo_launch_test_common.py

```diff
@@ -156,12 +166,15 @@ def generate_servo_test_description(
             ),
             ros2_control_node,
             test_container,
+            servo_container,
             TimerAction(period=2.0, actions=[servo_gtest]),
+            launch_testing.util.KeepAliveProc(),
```

Comment:
Good job figuring this out! I'm really impressed by your persistence with that.

Reply:
Thanks! I think this line helps but unfortunately without the sleep statements we still have some flakiness.